config/config.go

```diff
@@ -3,7 +3,14 @@ package config
 import "github.com/kelseyhightower/envconfig"
 
 type Configuration struct {
-	BindAddr string `envconfig:"BIND_ADDR"`
+	BindAddr    string `envconfig:"BIND_ADDR"`
+	MongoConfig MongoConfig
+}
+
+type MongoConfig struct {
+	BindAddr   string `envconfig:"MONGODB_BIND_ADDR"   json:"-"`
```

Comment:
Because you have left the json tag of hyphen (-) against the mongo BindAddr field, it will prevent any output (incl. logs) to not display this field. It is useful to use to hide sensitive information using the hyphen value when outputting the data yet allows the application to be able to access this info internally.

Worth reading this helpful blog about using tags - https://www.digitalocean.com/community/tutorials/how-to-use-struct-tags-in-go